app/src/main/java/ml/docilealligator/infinityforreddit/adapters/CustomizeThemeRecyclerViewAdapter.java

```diff
@@ -129,7 +129,6 @@ public int getItemCount() {
 
     public void setCustomThemeSettingsItem(ArrayList<CustomThemeSettingsItem> customThemeSettingsItems) {
         this.customThemeSettingsItems.clear();
-        notifyDataSetChanged();
```

Comment:
I thought every modification to the list should be followed by notifyXXXX?

Reply:
Not exactly. You do need to call `notifyXXX` to make the changes visible. But there's no difference between doing `A->B->C` and `A->C` if `B` will never be displayed.

Reply:
Cool. Thank you.